src/index.tsx

```diff
@@ -51,6 +52,7 @@ const app = createApp({
   initialState: window.__PRELOADED_STATE__,
   services: {
     archiveLoader: createArchiveLoader(archiveUrl),
+    bookConfigLoader: createBookConfigLoader(window.location.origin),
```

Comment:
this should be configured like other urls

Reply:
What exactly do you mean? via config?

Reply:
you can do it similar to `archiveUrl`, `buyPrintConfigUrl` etc. from this file

Reply:
@PiotrKozlowski 
I wonder if the path to `release.json` file should be placed in config or just move all the logic for creating the `fetchURL` to the `bookConfigLoader` gateway?

Reply:
I would stick to the same format as for other gateways, so you pass a `baseUrl` as you currently do, and `/rex/release.json` is added in `createBookConfigLoader` as it is now
the only change is to replace `window.location.origin` with a const `bookConfigUrl` that will be created with `assertDefined` in the same way like for example `buyPrintConfigUrl`

Reply:
Okay, but there is question if we would like to make the `BOOK_CONFIG_URL` from `config.js` to receive the url as:
- `BOOK_CONFIG_URL: window.location.origin`
- or `BOOK_CONFIG_URL: typeof(window) === 'undefined' ? 'https://openstax.org' : window.location.origin`

And the naming seems weird. If this property only contains the origin path, maybe it should be named the way showing it. To make it feel reusable.

However it still feels awkward form me to store it in config, but maybe I'm missing something :D

Reply:
@TomWoodward what do you think?